### ISSUE TITLE
_meta is ignored in script json

### ISSUE THREAD

**Issue by chizmw** [chizmw/botc-custom-script-json2pdf#1]
e.g. https://botc-scripts.azurewebsites.net/script/140/1.4.1

The behaviour hasn't been tested with this project, so things might even blow up!

```json
[{"id": "_meta", "logo": "https://i.ibb.co/y6xbjrC/Reptiles-logo.png", "name": "Reptiles!", "author": "Aero"}, {"id": "chef"}, {"id": "washerwoman"}, {"id": "librarian"}, {"id": "sailor"}, {"id": "general"}, {"id": "chambermaid"}, {"id": "snake_charmer"}, {"id": "flowergirl"}, {"id": "undertaker"}, {"id": "innkeeper"}, {"id": "philosopher"}, {"id": "fool"}, {"id": "tea_lady"}, {"id": "virgin"}, {"id": "saint"}, {"id": "sweetheart"}, {"id": "heretic"}, {"id": "drunk"}, {"id": "barber"}, {"id": "poisoner"}, {"id": "devils_advocate"}, {"id": "spy"}, {"id": "psychopath"}, {"id": "baron"}, {"id": "al-hadikhia"}, {"id": "storm_catcher"}, {"id": "djinn"}
```

**Comment by chizmw** [comment]
Confirming that this does cause the script to fail with an error:

```python
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/Users/chisel/development/github-as-chizmw/botc-custom-script-json2pdf/botcpdf/main.py", line 30, in <module>
    main()
  File "/Users/chisel/development/github-as-chizmw/botc-custom-script-json2pdf/botcpdf/main.py", line 24, in main
    script = Script(script_name, script_data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/chisel/development/github-as-chizmw/botc-custom-script-json2pdf/botcpdf/script.py", line 181, in __init__
    self.add_char(char)
  File "/Users/chisel/development/github-as-chizmw/botc-custom-script-json2pdf/botcpdf/script.py", line 228, in add_char
    role = self.role_data.get_role(char["id"])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/chisel/development/github-as-chizmw/botc-custom-script-json2pdf/botcpdf/script.py", line 93, in get_role
    return self.roles[id_slug]
           ~~~~~~~~~~^^^^^^^^^
KeyError: 'meta'
make[1]: *** [process] Error 1
```

**Comment by chizmw** [comment]
This should be resolved with #6
